.github/workflows/bintray.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish to Bintray
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  publish:
+    name: Publish to Bintray
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout
+    - name: Set up Java
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.11
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Upload SDK
+      env:
+        BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
+        BINTRAY_API_KEY: ${{ secrets.BINTRAY_API_KEY }}
+      run: BINTRAY_USER=$BINTRAY_USER BINTRAY_API_KEY=$BINTRAY_API_KEY ./gradlew bintrayUpload
```

Comment:
I dont think you would need to `BINTRAY_USER=$BINTRAY_USER BINTRAY_API_KEY=$BINTRAY_API_KEY`?

Reply:
Oh, ya. You're right. The env itself has been injected into that session right. I'll remove it